client/src/components/reviews/AddReview.jsx

```diff
@@ -11,87 +11,147 @@ class AddReview extends React.Component {
+          <input type='radio' id='sizeBig' name='size' value='4'></input>
+        </div>
+        <div className='radios'>
+          <label htmlFor='wide'>A size to wide</label>
+          <input type='radio' id='wide' name='size' value='5'></input>
+        </div>
       </div>);
     }
     if (description === 'Comfort') {
-      return (<div>
-        <label htmlFor='uncomfortable'>Uncomfortable</label>
-        <input type='radio' id='uncomfortable' name='comfort' value='1'></input>
-        <label htmlFor='sligthlyUncomfortable'>Sligthly Uncomfortable</label>
-        <input type='radio' id='sligthlyUncomfortable' name='comfort' value='2'></input>
-        <label htmlFor='ok'>Ok</label>
-        <input type='radio' id='ok' name='comfort' value='3'></input>
-        <label htmlFor='comfortable'>Comfortable</label>
-        <input type='radio' id='comfortable' name='comfort' value='4'></input>
-        <label htmlFor='comfortPerfect'>Perfect</label>
-        <input type='radio' id='comfortPerfect' name='comfort' value='5'></input>
+      return (<div className='reviewRadio'>
+        <div className='radios'>
+          <label htmlFor='uncomfortable'>Uncomfortable</label>
+          <input type='radio' id='uncomfortable' name='comfort' value='1'></input>
+        </div>
+        <div className='radios'>
+          <label htmlFor='sligthlyUncomfortable'>Sligthly Uncomfortable</label>
+          <input type='radio' id='sligthlyUncomfortable' name='comfort' value='2'></input>
+        </div>
+        <div className='radios'>
+          <label htmlFor='ok'>Ok</label>
+          <input type='radio' id='ok' name='comfort' value='3'></input>
+        </div>
+        <div className='radios'>
+          <label htmlFor='comfortable'>Comfortable</label>
+          <input type='radio' id='comfortable' name='comfort' value='4'></input>
+        </div>
+        <div className='radios'>
+          <label htmlFor='comfortPerfect'>Perfect</label>
+          <input type='radio' id='comfortPerfect' name='comfort' value='5'></input>
+        </div>
       </div>);
     }
     if (description === 'Length') {
-      return (<div>
-        <label htmlFor='short'>Runs Short</label>
-        <input type='radio' id='short' name='length' value='1'></input>
-        <label htmlFor='slightlyShort'>Runs slightly Short</label>
-        <input type='radio' id='slightlyShort' name='length' value='2'></input>
-        <label htmlFor='lengthPerfect'>Perfect</label>
-        <input type='radio' id='lengthPerfect' name='length' value='3'></input>
-        <label htmlFor='slightlyLong'>Runs slightly long</label>
-        <input type='radio' id='sligthlyLong' name='length' value='4'></input>
-        <label htmlFor='lengthLong'>Runs Long</label>
-        <input type='radio' id='lengthLong' name='length' value='5'></input>
+      return (<div className='reviewRadio'>
+        <div className='radios'>
+          <label htmlFor='short'>Runs Short</label>
```

Comment:
Comparing this version of code and the last one, I think this is so much cleaner! Great job!